shared/src/main/resources/config.yml

```diff
@@ -18,6 +18,14 @@
 # (!) IF YOU USE BUNGEE, Check & set on every Spigot server spigot.yml -> bungeecord: true.  (!)
 # (!) & Install Skinsrestorer.jar on ALL SERVERS!!! (BOTH Spigot & Bungeecord).              (!)
 
+##########
+# Locale #
+##########
+
+# A language code for the language you want to use by default for messages and commands.
+# Has to a string separated by an underscore.
+Language: en_US
```

Comment:
Would it be possible to make a "auto" mode in which it will detect  the server language

Reply:
Eh, I would rather not. Some people want to run their English Minecraft server on a Spanish host, for example. In my opinion, English is a good default setting.